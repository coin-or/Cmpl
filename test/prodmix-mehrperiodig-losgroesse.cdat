%P set < 1..2 >
%M set < 1..2 >
%J set < 1..3 >
	
%p[P,J] < 235 235 234
		255 253 250 >

%k[P,J] < 100 97 95
		 90 89  88 >

%kl[P,J] < 10 10 10
		  14 14 14 >

%a[M, P, J] < 	15 14 13
			25 25 25	
	       	25 22 20
			30 28 27>

%b[M,J] <	2500  1800 2200
		2500 3000 2000>

%xaMin[P,J] < 0 0 0 	0 10 20 >
%xaMax[P,J] < 40 50 50  40 70 80 >

%kr[P,J] < 500 490 480  450 450 450 >
%bigM  < 1000 >

%qMax[P,J] < 80 80 80  100 100 100  >
