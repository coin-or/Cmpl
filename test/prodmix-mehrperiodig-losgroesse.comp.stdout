---------------------------------------------------------------------------------------------------------
Problem              prodmix-mehrperiodig-losgroesse.cmpl
Nr. of variables     24
Nr. of constraints   18
Objective name       gdb
Solver name          CBC
Display variables    (q[*,y[*,x*)
Display constraints  (all)
---------------------------------------------------------------------------------------------------------

Objective status     optimal
Objective value      39029 (max!)

Variables 
Name                 Type            Activity         Lower bound         Upper bound            Marginal
---------------------------------------------------------------------------------------------------------
q[1,1]                  I                   0                   0                  80                   -
q[1,2]                  I                  50                   0                  80                   -
q[1,3]                  I                   0                   0                  80                   -
q[2,1]                  I                  43                   0                 100                   -
q[2,2]                  I                   0                   0                 100                   -
q[2,3]                  I                   0                   0                 100                   -
y[1,1]                  B                   0                   0                   1                   -
y[1,2]                  B                   1                   0                   1                   -
y[1,3]                  B                   0                   0                   1                   -
y[2,1]                  B                   1                   0                   1                   -
y[2,2]                  B                   1                   0                   1                   -
y[2,3]                  B                   1                   0                   1                   -
x[1,1]                  C                   0                   0                 inf                   -
x[1,2]                  C                 100                   0                 inf                   -
x[1,3]                  C                   0                   0                 inf                   -
x[2,1]                  C                  83                   0                 inf                   -
x[2,2]                  C                  16                   0                 inf                   -
x[2,3]                  C                  74                   0                 inf                   -
xa[1,1]                 I                   0                   0                  40                   -
xa[1,2]                 I                  50                   0                  50                   -
xa[1,3]                 I                  50                   0                  50                   -
xa[2,1]                 I                  40                   0                  40                   -
xa[2,2]                 I                  59                  10                  70                   -
xa[2,3]                 I                  74                  20                  80                   -
---------------------------------------------------------------------------------------------------------

Constraints 
Name                 Type            Activity         Lower bound         Upper bound            Marginal
---------------------------------------------------------------------------------------------------------
line_2                  L                2075                -inf                2500                   -
line_3                  L                1800                -inf                1800                   -
line_4                  L                1850                -inf                2200                   -
line_5                  L                2490                -inf                2500                   -
line_6                  L                2648                -inf                3000                   -
line_7                  L                1998                -inf                2000                   -
mBilanz[1,1]            E                   0                   0                   0                   -
x_y[1,1]                G                   0                   0                 inf                   -
mBilanz[1,2]            E                   0                   0                   0                   -
x_y[1,2]                G                 900                   0                 inf                   -
mBilanz[1,3]            E                   0                   0                   0                   -
x_y[1,3]                G                   0                   0                 inf                   -
mBilanz[2,1]            E         1.42109e-14                   0                   0                   -
x_y[2,1]                G                 917                   0                 inf                   -
mBilanz[2,2]            E                   0                   0                   0                   -
x_y[2,2]                G                 984                   0                 inf                   -
mBilanz[2,3]            E                   0                   0                   0                   -
x_y[2,3]                G                 926                   0                 inf                   -
---------------------------------------------------------------------------------------------------------
